src/UserSpaceInstrumentation/DummyProcessForTest.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "DummyProcessForTest.h"
+
+#include <chrono>
+#include <cstdio>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "OrbitBase/Logging.h"
+#include "absl/time/clock.h"
```

Comment:
I think we decided to use `<>` for `absl` headers (there are still many with `""` but iwyu will use `<>`). Here and in the other files.

Reply:
Why that? That seems wrong. 
<...> is intended for system headers and allows the implemtation to handle them differently from other stuff. The only guarantee you really get is that "..." also uses the same resolving mechanism as <...> does (if it can't find it by its own mechanism).

Reply:
Well, I don't have a strong opinion, I don't think it was for technical reasons, but more just to visually separate the includes of our code from includes of library code. I mentioned it because of consistency, especially if IWYU goes the other way.